docs/stream-aggregation.md

```diff
@@ -334,35 +336,128 @@ For example, the following config removes the `:1m_sum_samples` suffix added [to
-* `avg` returns the average input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `stddev` returns [standard deviation](https://en.wikipedia.org/wiki/Standard_deviation) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `stdvar` returns [standard variance](https://en.wikipedia.org/wiki/Variance) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `histogram_bucket` returns [VictoriaMetrics histogram buckets](https://valyala.medium.com/improving-histogram-usability-for-prometheus-and-grafana-bc7e5df0e350)
-  for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-* `quantiles(phi1, ..., phiN)` returns [percentiles](https://en.wikipedia.org/wiki/Percentile) for the given `phi*`
-  over the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
-  The `phi` must be in the range `[0..1]`, where `0` means `0th` percentile, while `1` means `100th` percentile.
-
 The aggregations are calculated during the `interval` specified in the [config](#stream-aggregation-config)
 and then sent to the storage.
 
 If `by` and `without` lists are specified in the [config](#stream-aggregation-config),
 then the [aggregation by labels](#aggregating-by-labels) is performed additionally to aggregation by `interval`.
 
+Below are aggregation functions that can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config).
+
+### total
+
+`total` generates output [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) by summing the input counters.
+`total` only makes sense for aggregating [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) type metrics.
+
+The results of `total` is equal to the `sum(some_counter)` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and `by: ["instance"]` and  the regular query:
+
+<img alt="total aggregation" src="stream-aggregation-check-total.png">
+
+### increase
+
+`increase` returns the increase of input [counters](https://docs.victoriametrics.com/keyConcepts.html#counter).
+`increase` only makes sense for aggregating [counter](https://docs.victoriametrics.com/keyConcepts.html#counter) type metrics.
+
+The results of `increase` with aggregation interval of `1m` is equal to the `increase(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and `by: ["instance"]` and  the regular query:
+
+<img alt="increase aggregation" src="stream-aggregation-check-increase.png">
+
+### count_series
+
+`count_series` counts the number of unique [time series](https://docs.victoriametrics.com/keyConcepts.html#time-series).
+
+The results of `count_series` is equal to the `count(some_counter)` query.
```

Comment:
```suggestion
The results of `count_series` is equal to the `sum(my_metric)` query.
```

`some_counter` implies it should be a counter. afaik, it could be anything in this case. Is my understanding correct?

Reply:
same for other examples below

Reply:
fixed